submissions/kolawoleOgunfowokan/HTML/js/script.js

```diff
@@ -0,0 +1,74 @@
+
+  let productArray = 
+[
+     { 
+       image: "../images/eutetic-freezer.jpg",
+       title: "Freezer",
+       price: "N195000"
+     },
+
+     {
+       image: "../images/huaweiMateBookXPro.jpg",
+       title: "Huwaei Laptop",
+       price: "N200000"
+     },
+
+     {
+       image: "../images/java-prog.png",
+       title: "Java Book",
+       price: "N12000"
+     },
+
+     {
+       image: "../images/microsoftSurface.jpg",
+       title: "Microsoft Surface",
+       price: "N320000"
+     },
+
+     {
+       image: "../images/introprog.jpg",
+       title: "Introduction to Programming",
+       price: "N10000"
+     },
+
+     {
+       image: "../images/Macbook.jpg",
+       title: "Macbook pro",
+       price: "N490000"
+     },
+
+     {
+       image: "../images/AppleDesktop.png",
+       title: "Apple Desktop",
+       price: "N350000"
+     },
+];
+
+let displayContent = document.getElementById("product-box");
```

Comment:
One last thing...
Generally in UX, elements have default behaviours. For example, one would expect a click event to be bound to a button, bot just any div. It took a while for me to figure out where to click to see your JS in action. Please use a button preferably

Reply:
Alright, finally, I got the button to display the products. I also was able to turn it off once it has been clicked, such that it doesn't keep populating the page unnecessarily. @kenny-wealth.